scripts/train.py

```diff
@@ -1,15 +1,30 @@
+import os
+from datetime import datetime
+import shutil
 from jsonargparse import CLI
 import json
 from dataclasses import asdict
 from src.config import BoldOrNotConfig
+from src.logging import setup_logging
 from src.model_training import train_model
+from src.plot import plot_training_curves
+
+project_path = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+current_date = datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
+current_training = f"training{current_date}"
```

Comment:
move to config,
exp_name = config["exp_name"]
current_training_name = f"{exp_name }_{current_date}"